lib/web_ui/dev/README.md

```diff
@@ -31,17 +31,34 @@ felt build [-w] -j 100
 If you are a Google employee, you can use an internal instance of Goma to parallelize your builds. Because Goma compiles code on remote servers, this option is effective even on low-powered laptops.
 
 ## Running web engine tests
-To run all tests:
+To run all tests on Chrome:
```

Comment:
nit: for consistency let's add one empty line before and after the code snippets

Reply:
done